src/compo/viirs_l1bnc2ioda.py

```diff
@@ -273,13 +301,21 @@ def main():
         help="percentage of random thinning fro 0.0 to 1.0. Zero indicates"
         " no thinning is performed. (default: %(default)s)",
         type=float, default=0.0)
+    parser.add_argument(
+        '-s', '--secterm',
```

Comment:
recommend removing the `-s` shortcut has been our standard practice as they can be too vague
```suggestion
        '--secterm',
```

Reply:
Sure, thanks. I have an impression that every argument is going to be in long name rather than the shortcut. @BenjaminRuston do you want me to update other arguments?

Reply:
yes please do so ...

see there is are two input files.... would say the `-o` is about the only option shortcut that could remain, and personally `obsinfo` and `geoinfo` would use `inputObs` and `inputGeo` as the long names but this is semantics and that change should be done separately from this PR 

so in short you could leave the output on `-o` but for the other remove the shortcut option name
also please see if my change works,, you can then merge into this branch... did you have a couple input files so I could run this too

Reply:
and I'm continuing to use your test files and that should be fine..

but did you notice in this program the channel number is not being recorded correctly:
```
>>> f.keys()
<KeysViewHDF5 ['Channel', 'Location', 'MetaData', 'ObsError', 'ObsValue', 'PreQC']>
>>> f['Channel'][:]
array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11], dtype=int32)
```

Reply:
wait this is fixed ;0)